auxiliary.py

```diff
@@ -0,0 +1,30 @@
+import os
+from matplotlib.cbook import get_sample_data
+import matplotlib.patches as mpatches
+import matplotlib.pyplot as plt
+
+
+uco = mpatches.Patch(color=(1., 1., 0.), label='UCO')
+buffer = mpatches.Patch(color=(1., 0.5, 0.), label='Buffer')
```

Comment:
consider a more descriptive name (perhaps the material? graphite? steel? helium?)